roles/image_builder/tasks/main.yml

```diff
@@ -7,6 +7,11 @@
     builder_custom_repos: "{{ microshift_image_custom_repos }}"
     builder_compose_pkgs: "{{ microshift_image_compose_pkgs }}"
 
+- name: Get rhsm repos properties
```

Comment:
probably need to wrap a conditional around this to make sure `microshift_rhsm_repos` is defined